docs/installation/autoware/docker-installation.md

```diff
@@ -77,7 +77,16 @@ You might need to log out and log back to make the current user able to use dock
    vcs import src < autoware.repos
    ```
 
-3. Build the workspace.
+3. Install missing dependencies.
+
+   ```bash
+   sudo apt update
+   source /opt/ros/galactic/setup.bash
+   rosdep update
+   rosdep install --from-paths . --ignore-src --rosdistro $ROS_DISTRO
+   ```
```

Comment:
How about this?

- Added some descriptions.
- I think `source /opt/ros/galactic/setup.bash` isn't necessary here because it's automatically sourced. (But it's okay to keep it.)

``````suggestion
3. Update dependent ROS packages.

    The dependency of Autoware may change after the Docker image was created.
    In that case, you need to run the following commands to update the dependency.

   ```bash
   sudo apt update
   rosdep update
   rosdep install --from-paths . --ignore-src --rosdistro $ROS_DISTRO
   ```
``````

Reply:
Thank you this is great :bow: